conf/app.routes

```diff
@@ -207,6 +207,9 @@ POST        /:srn/money-borrowed
 GET         /:srn/change-money-borrowed                                                 controllers.nonsipp.moneyborrowed.MoneyBorrowedController.onPageLoad(srn: Srn, mode: Mode = CheckMode)
 POST        /:srn/change-money-borrowed                                                 controllers.nonsipp.moneyborrowed.MoneyBorrowedController.onSubmit(srn: Srn, mode: Mode = CheckMode)
 
+GET         /:srn/what-you-will-need-money-borrowed                                     controllers.nonsipp.moneyborrowed.WhatYouWillNeedMoneyBorrowedController.onPageLoad(srn: Srn)
```

Comment:
Do you think it would be good to separate this out e.g. to create `moneyborrowed.routes` ?